api/src/main/java/org/openmrs/module/patientqueueing/utils/QueueingUtil.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs.module.patientqueueing.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class QueueingUtil {
+	
+	public static Date dateFormtterDate(Date date, String time) throws ParseException {
```

Comment:
I do not seem to understand this method name.

Reply:
There is another similar method that returns a string. Named these two basing on what they return

Reply:
What i mean is, when one just looks at a method named `dateFormtterDate`, are they able to tell what the method is to be used for?